packages/cerebral-website-tutorial/parts/05_update_state/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Controller } from 'cerebral'
+import App from './components/App'
+import { Container } from 'cerebral/react'
+import Devtools from 'cerebral/devtools'
+import { set } from 'cerebral/operators'
```

Comment:
cerebral-provider-http is now in monorepo so you can use it directly. 
I think your problem with setup is that Windows has problem with the ENV setting. For me setup also doesn't work, but it stops later when prepublishing 
TodoMVC.